pricegraph/src/num.rs

```diff
@@ -80,7 +80,13 @@ pub fn is_strictly_positive_and_finite(value: f64) -> bool {
 
 /// Returns true if an amount is considered a dust amount. See `MIN_AMOUNT`
 /// documentation for more details.
-pub fn is_dust_amount(amount: f64) -> bool {
+pub fn is_dust_amount_f(amount: f64) -> bool {
+    amount < MIN_AMOUNT as f64
+}
+
+/// Returns true if an amount is considered a dust amount. See `MIN_AMOUNT`
+/// documentation for more details.
+pub fn is_dust_amount_i(amount: u128) -> bool {
```

Comment:
Let me know if you prefer a different name for these functions. I *think* after the BigRational change we will only have 1 version again.

Reply:
Now that I think of it, maybe we should just make this
```
pub fn is_dust_amount(amount: u128) -> bool { ... }
```

And then replace calls with `is_dust_amount(some_f64_value as _)`.